src/error.rs

```diff
@@ -255,6 +280,29 @@ impl<Src, Dst: ?Sized> AlignmentError<Src, Dst> {
     }
 }
 
+impl<Src, Dst: ?Sized + Unaligned> From<AlignmentError<Src, Dst>> for Infallible {
+    #[inline(always)]
+    fn from(_: AlignmentError<Src, Dst>) -> Infallible {
```

Comment:
Do we care about documenting this? It's unlikely that callers will ever interact with this impl directly.

Reply:
Agreed.